### PR TITLE
Kf 2267 feat jupyter pytorch cuda full rock update

### PR DESCRIPTION
**Pull request by i-chvets**
Since this ROCK has been built during the workshopd and new features came out in the rockcraft, it requires a minor update.

Summary of changes:
- Updated with new non-root user setup.
- Updated security team requirements part.

### CODE REVIEW

**Review by misohu — APPROVED** [review]

File: jupyter-pytorch-cuda-full/rockcraft.yaml [hunk 1]
```diff
@@ -10,12 +10,12 @@ description: |
...
 license: Apache-2.0
 base: ubuntu:22.04
+run-user: _daemon_
 services:
   jupyter:
     override: replace
     summary: "jupyter-pytorch-cuda-full service"
     startup: enabled
-    user: jovyan
     environment:
```
Comment (SamirPS) [inline jupyter-pytorch-cuda-full/rockcraft.yaml:19]: why do you need this?

**Review by SamirPS — CHANGES_REQUESTED** [review]

File: jupyter-pytorch-cuda-full/rockcraft.yaml (hunk 1 above)
Comment (i-chvets) [reply jupyter-pytorch-cuda-full/rockcraft.yaml:19]: Removed.

Comment (i-chvets) [reply jupyter-pytorch-cuda-full/rockcraft.yaml:19]: Removed.

Comment (i-chvets) [reply jupyter-pytorch-cuda-full/rockcraft.yaml:19]: It is required by application.

**Pull request merged**